bin/helpers/utils.js

```diff
@@ -389,6 +390,42 @@ exports.setSpecTimeout = (bsConfig, args) => {
   logger.debug(`Setting spec timeout = ${specTimeout}`);
 }
 
+exports.isTimezoneArgPassed = () => {
+  return this.searchForOption('--timezone'); 
+}
+
+exports.isValidTimezone = (timezone) => {
+  if(!this.isUndefined(timezone) && !this.isUndefined(TIMEZONE[timezone])) {
+    return true;
+  }
+  return false;
+}
+
+exports.setTimezone = (bsConfig, args) => {
+  let timezone = undefined;
+  if(this.isTimezoneArgPassed()) {
```

Comment:
we can take a variable and save the timezone value passed with OR condition from args and browserstack.json and check check for validations. This will help to reduce redudant code